src/preview/rewriteStyleSheet.test.ts

```diff
@@ -243,41 +243,74 @@ describe("rewriteStyleSheet", () => {
 
   it('supports "::slotted"', () => {
     const sheet = new Sheet("::slotted(:hover) { color: red }")
-    rewriteStyleSheet(sheet as any)
+    rewriteStyleSheet(sheet as any, true)
     const selectors = sheet.cssRules[0].getSelectors()
     expect(selectors).toContain("::slotted(:hover)")
     expect(selectors).toContain("::slotted(.pseudo-hover)")
     expect(selectors).toContain(":host(.pseudo-hover-all) ::slotted(*)")
   })
 
   it('supports "::slotted" with classes', () => {
-    const sheet = new Sheet("::slotted(.a:hover, .b) .c { color: red }")
-    rewriteStyleSheet(sheet as any)
+    const sheet = new Sheet(".a > slot::slotted(.b:hover) { color: red }")
```

Comment:
I learned that `::slotted()` does not support a list of selectors as an argument, nor does it support and subsequent (e.g. descendant) selectors. I modified tests to avoid invalid use cases.